src/main/java/seedu/address/model/task/TaskIsInEventPredicate.java

```diff
@@ -0,0 +1,43 @@
+package seedu.address.model.task;
+
+import java.util.function.Predicate;
+
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.model.event.Event;
+
+/**
+ * Tests that a {@code Task} is linked to the given event.
+ */
+public class TaskIsInEventPredicate implements Predicate<Task> {
+    //Event to check if Person is involved.
```

Comment:
spacing would be good here between the `//` and Event

Reply:
Good catch!